stripe-test-e2e/src/test/java/com/stripe/android/test/e2e/CreatedPaymentIntent.kt

```diff
@@ -0,0 +1,11 @@
+package com.stripe.android.test.e2e
+
+import com.squareup.moshi.Json
```

Comment:
I haven't used moshi, I know you mentioned kotlin has a built in one as well?

Reply:
Good call. I'll see if there's support for that in Retrofit in a follow-up.